qa/L0_client_memory_growth/client_memory_mail.py

```diff
@@ -37,7 +37,7 @@
     today = date.today().strftime("%Y-%m-%d")
     subject = "Triton Client Memory Growth " + sys.argv[1] + " Summary: " + today
     memory_graphs = glob.glob("client_memory_growth*.log")
-    write_up = "<p>This test is run for both HTTP and GRPC protocols using C++ and Python test scripts. The max-allowed difference between mean and maximum memory usage is set to 10MB and 1MB for C++ and Python tests individually.</p>"
+    write_up = "<p>This test is run for both HTTP and GRPC protocols using C++ and Python test scripts for around 8 hours. The max-allowed difference between mean and maximum memory usage is set to 10MB and 1MB for C++ and Python tests individually.</p>"
```

Comment:
Why do we want to include the test duration? If we do it's better to call the approximate time for east test case instead of the overall runtime of the test

Reply:
David G suggests that it would be great if the test duration is included in the emails. Will change to showing the time for each test case.